### ISSUE TITLE
[FEATURE REQUEST] use offset in lower/upper bound

### ISSUE THREAD

**Issue by shift-del1** [kalkih/mini-graph-card#888]
Can you please add the option to have an offset in variable lower_bound or upper_bound? Sometimes the graph is too close to the border of the button, so I would like to shift it a bit. Defining a constant value (like 22.4 (%)) is not ok, as the humidity can vary in a huge range which could flatten the graph too much (like this happened in this case):

![image](https://user-images.githubusercontent.com/63366470/206923675-4b005e09-81da-4dc9-b451-66e170dac92e.png)

Using a string like:
lower_bound: "[[[ return states['sensor.kitchen_humidity_min_over_last_24_hours'].state - 2.5 ]]]"
could help, but defining a statistical sensor for all the rooms and measured values is very painful. So from this:

![image](https://user-images.githubusercontent.com/63366470/206923981-198d3995-987b-4ec4-983a-dfae9c46c0b8.png)

I would like to go to something like this:

![image](https://user-images.githubusercontent.com/63366470/206924178-8e7f31e6-984e-402e-844c-7b6e603fb704.png)

With a value formatted like ˇ2.5 (example prefix; take the min value of the used period (52.2)and lower it by 2.5) it could be achieved. Thank you.

**Comment by ildar170975** [comment]
Have you tested a `min_bound_range` option?
It may help you to achieve your goal.

**Comment by shift-del1** [comment]
> Have you tested a `min_bound_range` option? It may help you to achieve your goal.

Yes, but in this case I have to maintain a min value for everything if I want to use that value for the offset. For example: my kitchen temperature varies between 20 and 22 degrees for the last 24 hours, I need to take the min value and considering 3 degrees offset, I need to set the min_bound_range to "min(kitchen temperature) - offset" (so min(20...22) - 3 = 17). Having the offset option would simplify things of creating tens of new statistical sensors. Or do you have a better idea? Thanks in advance.

**Comment by ildar170975** [comment]
1. Have you tried to NOT specify `lower_bound`? Then the graph will use min & max values.
2. If you really need a proposed functionality - I would suggest you to prepare a complete algorithm for this functionality. Nobody will do it except you). The algorithm should be as detailed as possible. [Also, please read this](https://github.com/kalkih/mini-graph-card/blob/master/CONTRIBUTING.md).

**Comment by ildar170975** [comment]
No feedback for a long time.
Closing.
Please let me know if there is a need to reopen this.
